avalanche/models/pytorchcv_wrapper.py

```diff
@@ -0,0 +1,145 @@
+################################################################################
+# Copyright (c) 2021 ContinualAI.                                              #
+# Copyrights licensed under the MIT License.                                   #
+# See the accompanying LICENSE file for terms.                                 #
+#                                                                              #
+# Date: 1-05-2020                                                              #
+# Author(s): Eli Verwimp                                       #
+# E-mail: contact@continualai.org                                              #
+# Website: www.continualai.org                                                 #
+################################################################################
+
+"""
+This module wraps pytorchcv models, with some convient wrappers.
```

Comment:
Can you expand this a little with the ideas that you put already in the description of the PR? This may be useful to understand the scope of the module better!

Reply:
Alright, updated now!